io.neovim.nvim.tool.tmux.yml

```diff
@@ -0,0 +1,56 @@
+app-id: io.neovim.nvim.tool.tmux
+branch: stable
+runtime: io.neovim.nvim
+runtime-version: stable
+sdk: org.freedesktop.Sdk//22.08
+build-extension: true
+separate-locales: false
+appstream-compose: false
+
+modules:
+  - name: libevent
+    sources:
+      - type: archive
+        url: https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
+        sha256: 92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
+    buildsystem: simple
+    build-commands:
```

Comment:
this should be buildsystem: autotools.

Reply:
Fixed https://github.com/flathub/flathub/pull/4119/commits/893c32076c87ce0df66174823bef87486b9113d7